config/orchestrations/codeready-workspaces/0.1/codewind-clusterrole.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
I don't understand why this is a cluster role.  Che/CRW is installed to a single namespace, right?  Can this just be a Role that is bound to this one namespace?

What happens if I have a second Kabanero instance in a second namespace?  How are we preventing the ClusterRole from being deleted if the second Kabanero instance is deleted?  Wouldn't the first Che/CRW instance stop working?

Reply:
Ed explained that there is some logic in the Kabanero finalizer that will prevent the cluster role from being deleted if there is another Kabanero CR instance on the system.

John explained that Che expects this to be a cluster role, but we could manually create a role and a role binding (and ignore the cluster role setting on the CheCluster).

For the time being we are just going to leave it as it is.  When we start officially supporting two Kabanero instances on the system, if the Kabanero operator hasn't become a cluster scoped operator yet, we'll consider changing this to a Role.